### DISCUSSION TITLE
In "flet.app" is aviable parameter "permission".  What this is parameter and when do used?

### DISCUSSION THREAD

**Discussion by vist-amur** [flet-dev/flet#857] [Q&A]
In "flet.app" is aviable parameter "permission".  What this is parameter and when do used?

**Comment by ndonkoHenri** [comment]
@FeodorFitsner mentioned here: https://discord.com/channels/981374556059086931/981374557007015988/1040306164954570783 that this parameter is "not used anymore".

**Comment by vist-amur** [comment]
Thank you! Tell me please, how from the code to request permissions, for example, for do used clipboard or others?

**Reply by ndonkoHenri** [reply]
At the moment I dont think there's any stuffs to deal with device permissions. For the clipboard, you could just try to use it: https://flet.dev/docs/controls/page/#set_clipboarddata, and let us know if some extra permissions are required by the device running your app.
Maybe when Flet goes mobile, this cool dart/flutter package: https://pub.dev/packages/permission_handler could be intergrated.
